fili-core/src/test/groovy/com/yahoo/bard/webservice/web/JobsApiRequestSpec.groovy

```diff
@@ -121,63 +117,6 @@ class JobsApiRequestSpec extends Specification {
         errorSubscriber.getOnErrorEvents().get(0).getMessage() == "No job found with job ticket IDontExist"
     }
 
-    def "handleBroadcastChannelNotification returns an Observable that replays notification if it was sent before async timeout"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-
-        when:
-        Observable<PreResponse> preResponseObservable = defaultJobsApiRequest.handleBroadcastChannelNotification("ticket1")
-
-        broadcastChannel.publish("ticket1")
-        preResponseObservable.subscribe(testSubscriber)
-
-        then:
-        testSubscriber.assertReceivedOnNext([ticket1PreResponse])
-    }
-
-    def "handleBroadcastChannelNotification returns an Observable that filters notifications and only returns a single notification for the given ticket"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-
-        when:
-        Observable<PreResponse> preResponseObservable = defaultJobsApiRequest.handleBroadcastChannelNotification("ticket1")
-        broadcastChannel.publish("ticket_before_connecting_is_dropped")
-        broadcastChannel.publish("ticket1")
-        broadcastChannel.publish("ticket2")
-        broadcastChannel.publish("ticket1")
-        preResponseObservable.subscribe(testSubscriber)
-
-        then:
-        testSubscriber.assertReceivedOnNext([ticket1PreResponse])
-    }
-
-    def "handleBroadcastChannelNotification returns an empty Observable if a timeout occurs before the notification is received"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-        JobsApiRequest apiRequest = new JobsApiRequest(
-                null,
-                "0",
-                "",
-                "",
-                null,
-                uriInfo,
-                jobPayloadBuilder,
-                apiJobStore,
-                preResponseStore,
-                broadcastChannel
-        )
-
-        when: "We start listening to the broadcastChannel and timeout occurs"
-        Observable<PreResponse> preResponseObservable = apiRequest.handleBroadcastChannelNotification("ticket1")
-
-        and: "we subscribe to the preResponseObservable"
-        preResponseObservable.subscribe(testSubscriber)
-
-        then: "preResponseObservable is empty (the chain is complete, and no values were sent)"
-        ReactiveTestUtils.assertCompletedWithoutError(testSubscriber)
-        testSubscriber.assertNoValues()
-    }
```

Comment:
This is subsumed by tests in the `JobsServletReactiveChainforResultsEndpointSpec` that verify that the `getResults` method returns an empty Observable if the results are not ready within the asynchronous timeout.